cmd/gateway/commands.go

```diff
@@ -291,4 +297,42 @@ func createProvisionerModeCommand() *cobra.Command {
 			})
 		},
 	}
+
+	cmd.PersistentFlags().Var(
+		&gatewayCtlrName,
+		gatewayCtrlNameFlag,
```

Comment:
nit: can we change this to `gatewayCtlrNameFlag` for consistency with the other name? (ctrl vs ctlr)